packages/terra-demographics-banner/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* Added
+  * Added optional `personNameLevel` prop to set the person name header level with default of 2.
+  * Added optional `identifiersLongForm` prop to add well defined descriptions for shortened/abbreviated identifiers.
+
+* Changed
+  * Changed patient nickname before/after CSS parenthesis property to instead be in the string to improve screen reader user's understanding of visual presentation.
```

Comment:
Can this be streamlined a bit more? Maybe something like:

Improved representation of patient nickname for screenreaders by appending parentheses though a template string instead of CSS.

Reply:
Updated to suggested message here https://github.com/cerner/terra-core/pull/3787/commits/a6720f14168f9923339af29d8db9b5b431f0598c#diff-23b4f0bb288a072f5d6aa205a72db5a0affdcef4540c9d16ae1aa41e917e0461R10